app/views/admin/terms_of_service/previews/show.html.haml

```diff
@@ -0,0 +1,20 @@
+- content_for :page_title do
+  = t('admin.terms_of_service.preview.title')
+
+- content_for :heading_actions do
+  .back-link
+    = link_to admin_terms_of_service_index_path do
+      = material_symbol 'chevron_left'
+      = t('admin.terms_of_service.back')
+
+%p.lead
+  = t('admin.terms_of_service.preview.explanation_html', count: @user_count, display_count: number_with_delimiter(@user_count), date: l(@terms_of_service.published_at.to_date))
+
+.prose
+  = markdown(@terms_of_service.changelog)
+
+%hr.spacer/
+
+.content__heading__actions
+  = link_to t('admin.terms_of_service.preview.send_preview', email: current_user.email), test_admin_terms_of_service_preview_path(@terms_of_service), method: :post, class: 'button button-secondary'
```

Comment:
I feel like I suggested this elsewhere before and it was not embraced, but thoughts on using "lazy lookup" for these i18n keys? -- ie `t('.send_preview', ...)` here, and corresponding i18n key path?

Reply:
I don't have a super strong opinion on this but I am more used to writing it out explicitly.